### ISSUE TITLE
Path naming: avoid /sandbox

### ISSUE THREAD

**Issue by ObjectIsAdvantag** [DevNetSandbox/sbx_api_issues#6]
It's good practice to avoid brand names when designing an API, and go with meaningful name.
Imagine tomorrow the "DevNet Sandbox" name gets replaced by "DevNet VirtualDevices"

Then I'd suggest to replace the /sandbox path by /templates.

**Comment by ObjectIsAdvantag** [comment]
Then I'd suggest to replace the /sandbox path by /templates.

**Comment by TomDaviesUK** [comment]
I agree, but with the context we have here, it's both a 'brand name' and the 'thing a user reserves'.

My concern would be that the API would become less intuitive if we used 'templates', as we do not reference or discuss templates anywhere (or, what we actually use a little internally, blueprints).

Thoughts?

_Note: This would also mean updates to the corresponding parameters in the returned JSON, to state template_id rather than sandbox etc._

**Comment by ObjectIsAdvantag** [comment]
Terminology are the hardest.

I noticed the Sandbox UI mentions "Sandbox Labs" and "Labs Catalog"
Aligning the UI with the API is certainly a good option.
Then simply go with: api.sandbox.com/v1/labs
